sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/CreateTableAsSelect.scala

```diff
@@ -91,6 +93,6 @@ case class CreateTableAsSelect(
   }
 
   override def argString: String = {
-    s"[Database:$database, TableName: $tableName, InsertIntoHiveTable]\n" + query.toString
+    s"[Database:$database, TableName: $tableName, InsertIntoHiveTable]"
```

Comment:
I think it is fine to add the storage format in the string. But, `tableDesc` itself has too much information and it is hard to figure out the meaning.